src/Controller/EnMarche/AdherentMessage/ReferentMessageController.php

```diff
@@ -17,4 +17,9 @@ protected function getMessageType(): string
     {
         return AdherentMessageTypeEnum::REFERENT;
     }
+
+    public function disableInProduction(): void
+    {
+        // Override parent method for activating this feature in production
```

Comment:
You cannot remove the `$this->disableInProduction()` calls instead ?

Reply:
the call is in the parent class and I need to keep it for Deputy controller

Reply:
Ok I see, thx :+1: